wd_helper.au3

```diff
@@ -2180,6 +2180,57 @@ Func _WD_JsonActionPause($iDuration)
 	Return $sJSON
 EndFunc   ;==>_WD_JsonActionPause
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_JsonCookie
+; Syntax ........: _WD_JsonCookie($sName, $sValue[, $sPath = Default[, $sDomain = Default[, $bSecure = Default [,
+;                  $bHTTPOnly = Default[, $iExpiryTime = Default[, $sSameSite = Default]]]]]])
+; Parameters ....: $sName               - a string value. The name of the cookie.
+;                  $sValue              - a string value. The cookie value.
+;                  $sPath               - [optional] a string value. Default is '/'. This defines the cookie path.
+;                  $sDomain             - [optional] a string value. Default is '' - will be ommited and current browsing context’s of active document’s will be used. This defines the domain the cookie is visible to.
+;                  $bSecure             - [optional] a boolean value. Default is False. This defines whether the cookie is a secure cookie.
+;                  $bHTTPOnly           - [optional] a boolean value. Default is False. This defines whether the cookie is an HTTP only cookie.
+;                  $iExpiryTime         - [optional] an integer value. Default is 0 - will be ommited. This defines when the cookie expires, specified in seconds since Unix Epoch.
+;                  $sSameSite           - [optional] a string value. Default will be ommited and this mean "None". This defines whether the cookie applies to a SameSite policy. One of the following modes:
```

Comment:
Please update to conform with rest of UDF. Should be like

```autoit
;                  $sValue              - The cookie value.
;                  $sPath               - [optional] This defines the cookie path. Default is '/'. 
```

Reply:
Take a look: [_WD_JsonCookie() parameters descriptions](https://github.com/Danp2/au3WebDriver/pull/194/commits/30710af72b92ef28f2d5710205a07c59bdb48bc9)

Reply:
$sName & $sValue still needs to be done.

Reply:
Please check: [_WD_JsonCookie param descr fix](https://github.com/Danp2/au3WebDriver/pull/194/commits/6a24a523b393f1332088e6ea1db07afb657ebd76)